src/libs/style.js

```diff
@@ -83,33 +83,28 @@ function replaceSourceAccessToken(mapStyle, key, opts={}) {
 
   const changedSources = {
     ...mapStyle.sources,
-    [key]: {
+    [sourceName]: {
       ...source,
       url: source.url.replace('{key}', accessToken)
     }
   }
-  const changedStyle = {
+  let changedStyle = {
     ...mapStyle,
     sources: changedSources
   }
-
+  if (mapStyle.glyphs) {
+    changedStyle.glyphs = changedStyle.glyphs.replace('{key}', accessToken)
+  }
   return changedStyle
 }
 
 function replaceAccessTokens(mapStyle, opts={}) {
-  let changedStyle = mapStyle;
+  let changedStyle = mapStyle
 
-  Object.keys(mapStyle.sources).forEach((tokenKey) => {
-    changedStyle = replaceSourceAccessToken(changedStyle, tokenKey, opts);
+  Object.keys(mapStyle.sources).forEach((sourceName) => {
+    changedStyle = replaceSourceAccessToken(changedStyle, sourceName, opts);
   })
 
-  if(mapStyle.glyphs && mapStyle.glyphs.match(/\.tileserver\.org/)) {
```

Comment:
I think this just needs updating, see <https://github.com/openmaptiles/klokantech-basic-gl-style/blob/master/style.json#L19>

Reply:
don't you think it's better to replace single occurrences of `{key}` in the URL so that you don't have to maintain hard coded regexp?

Reply:
Ah I see you moved the code into `replaceSourceAccessToken`, my mistake